### PR TITLE
Update linear_pool_quantile.R

### PR DESCRIPTION
**Pull request by elray1**
change handling of data types for output_type_id in linear_pool_quantile

### CODE REVIEW

**Comment by elray1** [comment]
note, we should update the patch version of the package

File: tests/testthat/test-linear_pool.R [hunk 1]
```diff
@@ -329,23 +327,23 @@ test_that("(weighted) quantiles correctly calculated - lognormal family", {
                                           model_id = "hub-ensemble",
                                           task_id_cols = NULL,
                                           n_samples = 1e5) |>
-                            dplyr::mutate(output_type_id = as.numeric(output_type_id))
+                            dplyr::mutate(output_type_id = (output_type_id))
```
Comment (elray1) [inline tests/testthat/test-linear_pool.R:330]: Is this line doing anything, or could we just delete it?

File: tests/testthat/test-linear_pool.R [hunk 2]
```diff
@@ -329,23 +327,23 @@ test_that("(weighted) quantiles correctly calculated - lognormal family", {
...
 
   weighted_quantile_actual_norm <- linear_pool(weighted_component_outputs,
                                           weights = fweight1,
                                           weights_col_name = "weight",
                                           model_id = "hub-ensemble",
                                           task_id_cols = NULL,
                                           n_samples = 1e5) |>
-                                      dplyr::mutate(output_type_id = as.numeric(output_type_id))
+                                      dplyr::mutate(output_type_id = (output_type_id))
```
Comment (elray1) [inline tests/testthat/test-linear_pool.R:338]: same question as above

File: tests/testthat/test-linear_pool.R [hunk 3]
```diff
@@ -329,23 +327,23 @@ test_that("(weighted) quantiles correctly calculated - lognormal family", {
...
 
   quantile_actual_lnorm <- linear_pool(component_outputs, weights = NULL,
                                           weights_col_name = NULL,
                                           model_id = "hub-ensemble",
                                           task_id_cols = NULL,
                                           tail_dist = "lnorm",
                                           n_samples = 1e5) |>
-                              dplyr::mutate(output_type_id = as.numeric(output_type_id))
+                              dplyr::mutate(output_type_id = (output_type_id))
```
Comment (elray1) [inline tests/testthat/test-linear_pool.R:346]: and again

File: tests/testthat/test-linear_pool.R [hunk 4]
```diff
@@ -354,7 +352,7 @@ test_that("(weighted) quantiles correctly calculated - lognormal family", {
                                           task_id_cols = NULL,
                                           tail_dist = "lnorm",
                                           n_samples = 1e5) |>
-                                      dplyr::mutate(output_type_id = as.numeric(output_type_id))
+                                      dplyr::mutate(output_type_id = (output_type_id))
```
Comment (elray1) [inline tests/testthat/test-linear_pool.R:355]: and again

File: tests/testthat/test-linear_pool.R (hunk 1 above)
Comment (lshandross) [reply tests/testthat/test-linear_pool.R:330]: Was trying to get rid of the coercion to numeric type for the `output_type_id` here and in other instances but it looks like I didn't properly delete the entire mutate statement. I've fixed this in a new commit

**Pull request merged**